unit-tests/live/d500/test-temperatures-xu-vs-hwmc.py

```diff
@@ -0,0 +1,97 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# test:device D500*
+
+import pyrealsense2 as rs
+from rspy import test
+
+# This test checks that the same values of temperature are received whether XU command or HWM Command are used.
+
+dev = test.find_first_device_or_exit()
+depth_sensor = dev.first_depth_sensor()
+dp_device = dev.as_debug_protocol()
+
+
+########################################  HELPERS  ##########################################
+
+def get_temperatures_from_xu():
+    pvt_temp = -10
+    ohm_temp = -10
+    proj_temp = -10
+
+    test.check(depth_sensor.supports(rs.option.soc_pvt_temperature))
+    test.check(depth_sensor.supports(rs.option.ohm_temperature))
+    test.check(depth_sensor.supports(rs.option.projector_temperature))
+
+    pvt_temp = depth_sensor.get_option(rs.option.soc_pvt_temperature)
+    ohm_temp = depth_sensor.get_option(rs.option.ohm_temperature)
+    proj_temp = depth_sensor.get_option(rs.option.projector_temperature)
+
+    return pvt_temp, ohm_temp, proj_temp
+
+
+def parse_temperature_from_hwm(hwm_answer):
+    temperature = -10.0
+    relevant_data = hwm_answer[4:]
+    temperatures_list = []
+
+    test.check_equal(len(relevant_data), 20)
+    whole_number_part = 0
```

Comment:
Can you add a comment explaining how the GTEMP data is expected to look like?
Great test!

Reply:
ok